package.json

```diff
@@ -70,32 +70,22 @@
     "@testing-library/jest-dom": "^5.16.3",
     "@testing-library/react": "^10.4.3",
     "@testing-library/user-event": "^12.0.11",
+    "@types/carbon-components-react": "^7.55.2",
     "@types/jest": "^24.9.1",
+    "@types/webpack-env": "^1.15.1",
+    "@typescript-eslint/parser": "^4.26.0",
+    "babel-eslint": "^10.0.2",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.2.4",
     "babel-preset-minify": "^0.5.1",
-    "jest": "^24.9.0",
-    "jest-cli": "^24.9.0",
-    "openmrs": "^3.2.1-pre.1055",
-    "@types/webpack-env": "^1.15.1",
-    "fork-ts-checker-webpack-plugin": "^6.3.3",
     "clean-webpack-plugin": "^3.0.0",
-    "webpack": "^5.37.0",
-    "webpack-cli": "^4.7.0",
-    "webpack-dev-server": "^4.2.1",
-    "husky": "^3.0.4",
     "concurrently": "^6.3.0",
-    "prettier": "^1.18.2",
-    "pretty-quick": "^1.11.1",
     "eslint": "^7.10.0",
-    "@typescript-eslint/parser": "^4.26.0",
-    "babel-eslint": "^10.0.2",
     "eslint-config-prettier": "^6.13.0",
     "eslint-config-ts-react-important-stuff": "^3.0.0",
     "eslint-plugin-jsx-a11y": "^6.5.1",
     "eslint-plugin-prettier": "^3.1.4",
     "eslint-plugin-react-hooks": "^4.3.0",
-    "identity-obj-proxy": "^3.0.0",
-    "rxjs": "^6.6.3"
```

Comment:
Is there a reason u removed these dependencies? Please revert, we'll clean this up after the upgrades. 

Tests are also failing because of the missing dependency `prettier` which u removed

Reply:
@pirupius Done

Reply:
@ODORA0 u have failing tests on the components. check the CI

Reply:
@pirupius Done, Builds have passed